### DISCUSSION TITLE
Letting Houdini See it!

### DISCUSSION THREAD

**Discussion by BSalem** [AcademySoftwareFoundation/openvdb#1327] [General]
I compiled and installed OpenVDB against Houdini 18.5, but I failed by anyway to let houdini see the installed files!
to the degree I brute forced installing inside the houdini factory tree, and I still cannot see any vdb nodes in houdini, as well as that I don't see any errors when houdini is loading

Note: libopenvdb is also in the LD_LIBRARY_PATH, still no luck (I delete all old vdb SOPs from houdini path - doesn't make any difference except that the factory vdbs gone as expected)

**Comment by Idclip** [comment]
One thing you could check is to set `HOUDINI_DSO_ERROR` when launching Houdini. You can use this to confirm whether Houdini is either "missing" the VDB installation or actually finds it but refuses to load it due to symbol issues:

https://www.sidefx.com/docs/houdini/ref/env.html

For example you could launch Houdini with:
```
 > export HOUDINI_DSO_ERROR=1
 > houdini
```

If you see no VDB related errors then you can also try launching with `HOUDINI_DSO_ERROR=2|3|4` to see what locations are being searched.
